festim/meshing/mesh_from_vertices.py

```diff
@@ -24,17 +24,19 @@ def __init__(self, vertices, **kwargs) -> None:
 
     def generate_mesh_from_vertices(self):
         """Generates a 1D mesh"""
-        vertices = sorted(np.unique(self.vertices))
-        nb_points = len(vertices)
-        nb_cells = nb_points - 1
-        editor = f.MeshEditor()
         mesh = f.Mesh()
-        editor.open(mesh, "interval", 1, 1)  # top. and geom. dimension are both 1
-        editor.init_vertices(nb_points)  # number of vertices
-        editor.init_cells(nb_cells)  # number of cells
-        for i in range(0, nb_points):
-            editor.add_vertex(i, np.array([vertices[i]]))
-        for j in range(0, nb_cells):
-            editor.add_cell(j, np.array([j, j + 1]))
-        editor.close()
+        if f.MPI.comm_world.rank == 0:
```

Comment:
I would add this for future reference.
```suggestion
        # the following is needed to avoid breaking in parrallel
        # see issue 497
        if f.MPI.comm_world.rank == 0:
```